src/services/ParaSwap.ts

```diff
@@ -84,18 +84,24 @@ export class ParaSwap implements ISwapProvider {
 
         const resJson = await fetch(`${this.baseUrl}/prices?${query}`)
         const response = resJson.json()
+
         return response
       })
     })
 
     const amounts = await Promise.all(requests)
 
+    let transactionError
```

Comment:
You should add the type just to be sure, since you're not initializing it directly

```suggestion
    let transactionError: string
```